src/__tests__/WelcomePage.test.tsx

```diff
@@ -1,13 +1,18 @@
 import React from "react";
-import ReactDOM from "react-dom";
+import { createRoot } from "react-dom/client";
 import WelcomePage from "../components/WelcomePage";
 import { cleanup } from "@testing-library/react";
-
+import { BrowserRouter as Router } from "react-router-dom";
 afterEach(cleanup);
 
 describe("Welcome Page", () => {
   it("renders without crashing", () => {
     const div = document.createElement("div");
-    ReactDOM.render(<WelcomePage start={undefined} />, div);
+    const root = createRoot(div!);
+    root.render(
+      <Router>
+        <WelcomePage />
```

Comment:
This component has a required prop so that will need to be added here to remove the error

Reply:
I removed this prop  , since it no more needed because of router